### PR TITLE
ansible-cmdb: init at 1.31, python3Packages.jsonxs: init at 0.6

### PR DESCRIPTION
**Pull request by tie**
## Description of changes

Packages http://github.com/fboender/ansible-cmdb

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by tie** [comment]
Result of `nixpkgs-review pr 317405` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>ansible-cmdb</li>
    <li>ansible-cmdb.dist</li>
    <li>python311Packages.jsonxs</li>
    <li>python311Packages.jsonxs.dist</li>
    <li>python312Packages.jsonxs</li>
    <li>python312Packages.jsonxs.dist</li>
  </ul>
</details>

File: pkgs/by-name/an/ansible-cmdb/package.nix [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    pyyaml
+    jsonxs
+    buildPythonApplication;
+
+  pname = "ansible-cmdb";
+  version = "1.31";
+in
+buildPythonApplication {
+  inherit pname version;
+  format = "setuptools";
```
Comment (felbinger) [inline pkgs/by-name/an/ansible-cmdb/package.nix:20]: `format` is deprecated, [to my knowledge](https://github.com/NixOS/nixpkgs/issues/253154). Use `pyproject = false` instead

**Comment by felbinger** [comment]
I got some comments earlier that the pr titel and commits should include `python3Packages.`, so I suggest updating your pr title to: `python3Packages.jsonxs: init at 0.6; python3Packages.ansible-cmdb: init at 1.31`

**Comment by felbinger** [comment]
It can also be build on `x86_64-linux`

Result of `nixpkgs-review pr 317405` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>ansible-cmdb</li>
    <li>ansible-cmdb.dist</li>
    <li>python311Packages.jsonxs</li>
    <li>python311Packages.jsonxs.dist</li>
    <li>python312Packages.jsonxs</li>
    <li>python312Packages.jsonxs.dist</li>
  </ul>
</details>

File: pkgs/by-name/an/ansible-cmdb/package.nix (hunk 1 above)
Comment (tie) [reply pkgs/by-name/an/ansible-cmdb/package.nix]: Fixed.

**Comment by felbinger** [comment]
> > I got some comments earlier that the pr titel and commits should include python3Packages., so I suggest updating your pr title to: python3Packages.jsonxs: init at 0.6; python3Packages.ansible-cmdb: init at 1.31 and you'r second commit to python3Packages.ansible-cmdb: init at 1.31
> 
> The `ansible-cmdb` package is not in the `python3Packages` package set (it’s an application, not python package), so it shouldn’t be prefixed. W.r.t. PR title, it’s the first time I’m hearing about this convention and I don’t think it’d scale to a more complex change sets, but I guess it’s fine since there are only two small init commits.

I see, my bad I oversaw this. I agree but I got these comments earlier on one of my pr's.

File: pkgs/by-name/an/ansible-cmdb/package.nix [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{ lib
+, fetchFromGitHub
+, substituteAll
+, python3
```
Comment (JohnRTitor) [inline pkgs/by-name/an/ansible-cmdb/package.nix:4]:
```suggestion
, python3Packages
```

File: pkgs/by-name/an/ansible-cmdb/package.nix [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+, testers
+, ansible-cmdb
+}:
+let
+  inherit (python3.pkgs)
```
Comment (JohnRTitor) [inline pkgs/by-name/an/ansible-cmdb/package.nix:9]:
```suggestion
  inherit (python3Packages)
```

**Review by JohnRTitor — COMMENTED** [review]
Additionally please format your files with `nixfmt-rfc-style`

**Comment by tie** [comment]
Done.

**Comment by JohnRTitor** [comment]
Result of `nixpkgs-review pr 317405` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>ansible-cmdb</li>
    <li>ansible-cmdb.dist</li>
    <li>python311Packages.jsonxs</li>
    <li>python311Packages.jsonxs.dist</li>
    <li>python312Packages.jsonxs</li>
    <li>python312Packages.jsonxs.dist</li>
  </ul>
</details>

**Review by JohnRTitor — APPROVED** [review]
LGTM

**Pull request merged**